Orange/tests/test_sparse_reader.py

```diff
@@ -7,7 +7,7 @@
 
 import numpy as np
 
-import Orange.data._io as _io
+import Orange.data._io as _io  # pylint: disable=useless-import-alias
```

Comment:
Why does it have to be imported like this? (and not from Orange.data import _io)